### ISSUE TITLE
Type declaration not found

### ISSUE THREAD

**Issue by RHOOPH** [TechStark/react-dnd-scrolling#27]
I get this error when using in typescript project. 

``` 
Could not find a declaration file for module 'react-dnd-scrolling'. '/var/home/user/Work/Repo/app-name/node_modules/.pnpm/react-dnd-scrolling@1.3.3_react-dnd@16.0.1_react-dom@18.2.0_react@18.2.0/node_modules/react-dnd-scrolling/lib/esm/index.js' implicitly has an 'any' type.
  There are types at '/var/home/user/Work/Repo/app-name/node_modules/react-dnd-scrolling/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'react-dnd-scrolling' library may need to update its package.json or typings.
  ```

**Comment by ttt43ttt** [comment]
https://stackoverflow.com/questions/58990498/package-json-exports-field-not-working-with-typescript
@RHOOPH could you try with TypeScript 4.7 or above?

**Comment by RHOOPH** [comment]
My typescript version is 5.4, I don't think that's the issue, 

I created a fresh [vite app](https://vitejs.dev/guide/#scaffolding-your-first-vite-project) with `react-ts` template

then added this package and imported in `App.tsx`, 

I still got the same error.  Setting moduleResolution to "NodeNext" or "Node16" also didn't help.

**Comment by ttt43ttt** [comment]
@RHOOPH I see. This is fixed in version 1.3.7
